docs/getting_started_on_linux.md

```diff
@@ -68,6 +68,42 @@ environment variable is set when you are building. You may find it useful to add
 `source [PATH TO VULKAN SDK]/setup-env.sh` to your `~/.bashrc` file to simplify
 environment variable setup.
 
+## Optional: Configure Git
```

Comment:
It sounds a bit weird to call this "optional" since you definitely need git for everything :-D

I see this mirrors the windows instructions.

Reply:
Should I leave this as is then?